docker/Dockerfile.mdbook

```diff
@@ -1,4 +1,4 @@
-FROM sphinxc0re/holochain-rust
+FROM holochain/holochain-rust:develop
```

Comment:
@sphinxc0re actually, this is what fixes the mdbook docker command and it's already in this PR, unless you want to implement a different solution.  #178

Reply:
No, that's fine. That's exactly how I would've fixed it